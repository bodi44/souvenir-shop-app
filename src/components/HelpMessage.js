import React from 'react'

const HelpMessage = () => (
  <div>
    <p>
      <strong>purchase &lt;date&gt; &lt;price&gt; &lt;currency&gt; &lt;productName&gt;</strong> - add purchase to
      store.<br/>
      <i>
        Example: purchase 2019-04-25 12 USD Photo <br/>
        Output: <br/>
        2019-04-25 <br/>
        Photo 12 USD
      </i>
    </p>
    <p>
      <strong>all</strong> - show all purchases. <br/>
      <i>
        Example: all <br/>
        Output: <br/>
        2019-04-25 <br/>
        Photo 12 USD
      </i>
    </p>
    <p>
      <strong>clear &lt;date&gt;</strong> - removes all purchases for specified date. <br/>
      <i>
        Example: clear 2019-04-25 <br/>
        Output:
      </i>
    </p>
    <p>
      <strong>report &lt;year&gt; &lt;currency&gt;</strong> - calculate the total income for specified year,
      convert
      and present it in specified currency. <br/>
      <i>
        Example: report 2019 UAH <br/>
        Output: <br/>
        356.4 UAH
      </i>
    </p>
  </div>
)

export default HelpMessage