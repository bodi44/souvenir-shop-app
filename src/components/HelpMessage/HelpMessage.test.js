import React from 'react'
import { shallow, configure } from 'enzyme'
import expect from 'expect'
import Adapter from 'enzyme-adapter-react-16'

import HelpMessage from '../HelpMessage'

configure({ adapter: new Adapter() })

describe('HelpMessage component', () => {
    it('should render correctly', () => {
      const component = shallow(<HelpMessage/>)
      expect(component).toMatchSnapshot()
    })
  }
)