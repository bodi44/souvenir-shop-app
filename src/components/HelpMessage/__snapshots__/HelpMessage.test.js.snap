// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpMessage component should render correctly 1`] = `ShallowWrapper {}`;
